app/code/Magento/Quote/Model/GuestCart/GuestCartTotalRepository.php

```diff
@@ -8,7 +8,6 @@
 
 use Magento\Quote\Api\CartTotalRepositoryInterface;
 use Magento\Quote\Api\GuestCartTotalRepositoryInterface;
-use Magento\Quote\Model\QuoteIdMask;
```

Comment:
We can't remove this line because `Magento\Quote\Model\QuoteIdMask` used in `get` method without the fully qualified name